docs/integrations/saas-cloud/druva.md

```diff
@@ -0,0 +1,117 @@
+---
+id: druva
+title: Druva
+sidebar_label: Druva
+description: The Druva App for Sumo Logic provides real-time monitoring and security insight for Druva accounts, enabling prompt detection and response to potential security incidents through granular insights into user activity, alerts, and backup status.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="100"/>
+
+The Druva App for Sumo Logic enables real-time monitoring and security insight for your Druva account. This app provides critical event alerts and visibility into user activities, device and app usage, and backup and restore events. With this level of visibility, security teams can quickly detect and respond to potential security incidents. The Druva App for Sumo Logic is an essential tool for organizations looking to enhance their security operations.
+
+The Druva App for Sumo Logic leverages Sumo Logic's cloud-to-cloud Druva source to collect and analyze data from your Druva account. This app delivers granular insights into user activity, alerts, and backup status, enabling organizations to promptly identify and respond to potential security incidents.
+
+The Druva App offers the following capabilities:
+
+* Monitor user activity and identify any unauthorized access to sensitive data.
+* Detect and mitigate potential ransomware attacks.
+* Ensure compliance with regulatory mandates.
+* Monitor endpoint security and identify potential threats.
+* Orchestrate incident response workflows by integrating with other security tools.
+
+The Sumo Logic Druva App is a powerful tool for monitoring and protecting your data backups and archives. Organizations may gain important insights into their data security posture and respond quickly and efficiently to possible security events by leveraging Sumo Logic's Events API and advanced analytics capabilities.
+
+## Log Types
+
+The Druva App utilizes Sumo Logic's Druva Source to gather [Audit Events](https://developer.druva.com/docs/event-apis) from Druva. For more information on the fields of inSync events, refer to the [inSync events fields](https://developer.druva.com/docs/event-apis#obtain-events-api-response-in-syslog-format) documentation.
+
+## Sample log message
```

Comment:
Sample Log Message
(Title case (initial cap) all doc titles)

Reply:
Please refer to the comment about capitalization.